hooknode/services/s3.go

```diff
@@ -0,0 +1,21 @@
+package services
+
+import "github.com/oysterprotocol/lambda-node/hooknode/types"
+
+type S3 struct{}
+
+// ChunkStore interface
+func (i *S3) AdaptReqChunks(chunks []types.ReqChunk) []types.IotaChunk {
+	result := make([]types.IotaChunk, len(chunks))
+	// TODO: Fetch Message from s3
+	for i, reqChk := range chunks {
+		result[i] = types.IotaChunk{
+			Address: reqChk.Address,
```

Comment:
If we're storing the hash in s3, would the address be null as well, prior to fetching?

Reply:
Yep, going to make all those changes in a future PR. Wanted this one to be just a refactoring.

Reply:
Okay sounds good